src/ApolloClient/client.js

```diff
@@ -1,8 +1,11 @@
 import {ApolloClient, InMemoryCache} from '@apollo/client';
-const client = new ApolloClient
-({
-  uri: 'https://fff-be-2e7913919a6b.herokuapp.com/graphql',
-  // uri: 'http://127.0.0.1:6969/graphql',
+
```

Comment:
This line imports necessary modules from the Apollo Client library. ApolloClient is the main class representing the GraphQL client, and InMemoryCache is used for caching GraphQL query results in memory.